python/setup.py.in

```diff
@@ -104,6 +105,35 @@ def mkl():
 write_version_py(filename='@PADDLE_BINARY_DIR@/python/paddle/version.py')
 
 
+def write_distributed_training_mode_py(filename='paddle/fluid/incubate/fleet/parameter_server/version.py'):
```

Comment:
when  is_transpiler() == False,  it means using pslib ?  it may not take into consider some cases like no transpile and no pslib

Reply:
I will fix it, thanks.